web/components/src/List/List.tsx

```diff
@@ -15,6 +15,7 @@ const StyledList = styled(EdsList)<ListProps>`
   margin-left: var(--space-medium);
   margin-right: var(--space-medium);
   padding-left: var(--space-medium);
+  padding-bottom: var(--space-small);
```

Comment:
Adding the padding to the List component itself will have a colateral to all these components:

![image](https://github.com/equinor/energyvision/assets/23337266/edf73481-0dae-431b-8698-d4d7d1a03459)

A solution like the one implemented for News is more suitable for this